Mention Cyclonus for NetworkPolicy validation (#1068)
fluxcd/website: content/en/docs/security/best-practices.md

    <summary>Audit Procedure</summary>

    - Check whether you adhere to [Kubernetes Network Isolation Guidelines](https://kubernetes.io/docs/concepts/security/multi-tenancy/#network-isolation)

>     - Create a [Network Policy](https://fluxcd.io/docs/flux-e2e/#fluxs-default-configuration-for-networkpolicy) and confirm it is being enforced by the CNI. Alternatively, run a tool such as [Cyclonus](https://github.com/mattfenwick/cyclonus) for validating NetworkPolicy enforcement by the CNI plugin on your cluster.

Reviewer: Maybe?

```suggestion
    - Confirm that the [Network Policy](https://fluxcd.io/docs/flux-e2e/#fluxs-default-configuration-for-networkpolicy) objects created by Flux are being enforced by the CNI. Alternatively, run a tool such as [Cyclonus](https://github.com/mattfenwick/cyclonus) or [Sonobuoy](https://github.com/vmware-tanzu/sonobuoy) to validate NetworkPolicy enforcement by the CNI plugin on your cluster.
```
